packages/flutter_tools/test/integration.shard/test_data/legacy_settings_gradle_project.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'basic_project.dart';
+import 'deferred_components_config.dart';
+import 'deferred_components_project.dart';
+
+class LegacySettingsGradleProject extends BasicProject {
```

Comment:
Legacy tends to be a bad name to use for variables given its meaning can change over time and in long lived projects there are frequently many legacy ways of doing things. Instead try to think of a name that describes this way of gradle orignization.

Reply:
+1